cmd/ipfs/daemon.go

```diff
@@ -92,7 +92,8 @@ do this by setting headers on the API.HTTPHeaders and Gateway.HTTPHeaders
 keys:
 
 	ipfs config --json API.HTTPHeaders.X-Special-Header '["so special :)"]'
-	ipfs config --json Gateway.HTTPHeaders.X-Special-Header '["so special :)"]'
+	ipfs config --json Gateway.HTTPHeaders.X-Special-Header '["so special
+	:)"]'
 
```

Comment:
No problem, there were a few places like this throughout the docs I can revert back.